### PR TITLE
Add support for service url in svc describe output

### LINKED ISSUE #907: Add "-o url" to "kn service describe" and "kn revision describe"
### Feature request

Having a dedicated command to only show the  URL is very helpful in a scripted context so that one does not have to know the specifics of the Service schema to extract the URL via a jsonpath.

### Use case

Retrieve the URL to process it further in a script.

### UI Example

```
url=$(kn service describe myservice -o url)
```

### PR DESCRIPTION
**Pull request by hemanrnjn**
## Changes

* Added Support for service url display in `kn service describe` as output format

## Reference

Fixes #907

### CODE REVIEW

**Comment by hemanrnjn** [comment]
@googlebot I consent

**Comment by hemanrnjn** [comment]
@googlebot I consent

**Comment by navidshaikh** [comment]
@googlebot rescan

**Comment by hemanrnjn** [comment]
@googlebot I fixed it.

**Comment by hemanrnjn** [comment]
@googlebot rescan

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -18,6 +18,10 @@
 |===
 | | Description | PR
 
+| 🎁
+| Add "url" output format to return service url in service describe
```
Comment (mattmoor) [inline CHANGELOG.adoc:22]: Remove trailing whitespace:
```suggestion
| Add "url" output format to return service url in service describe
```

**Comment by hemanrnjn** [comment]
> Thanks, looks good to me !
> 
> Some minor requests:
> 
> * Please add one line to the CHANGELOG.adoc describing the change.
> * I'm not sure if this is easy possible, but it would be awesome if we got get the new option value into `kn service describe --help`, too. Otherwise it's hard to find out for a user that this option exists. I'm aware that the machinery comes from a dependency, but maybe there's a hook into the help message ?

Both changes done.

**Pull request merged**